pkg/operator/operator.go

```diff
@@ -46,27 +46,26 @@ import (
 )
 
 var OperatorClient client.ClientInterface
+var operatorStore store.Store
```

Comment:
Why make it a global variable instead of calling GetStore whenever it's needed?

Reply:
Looking for a way to have the operator use the store that is passed in when initialized, which would help a lot with unit testing.  How about if we convert the operator to a struct that implements the `Start` and `Shutdown` method.  This way a client and store can be passed in when creating and there won't be a need for any of these globals?  Something like:
```
o := operator.New(client, store, etc...)
o.Start()
defer o.Shutdown()
```